common/meta/raft-store/src/state_machine/sm.rs

```diff
@@ -574,7 +584,12 @@ impl StateMachine {
         tracing::debug!("applied UpsertKV: {} {:?}", key, result);
 
         if let Some(subscriber) = &self.subscriber {
-            let _ = subscriber.kv_changed(&key_str, &prev, &result);
+            let update = match value_op {
+                Operation::Update(_v) => UpdateType::UpInsert,
+                Operation::Delete => UpdateType::Delete,
+                Operation::AsIs => UpdateType::UpInsert,
+            };
+            let _ = subscriber.kv_changed(&key_str, update, &prev, &result);
```

Comment:
I unresolved this comment, because if using `Some(x), None` to represent a `delete` operation, the `UpdateType` can be removed, but it is still there.

I do not quite get it. Are you gonna remove `UpdateType` or still use it?